airflow/providers/google/cloud/operators/sql_to_gcs.py

```diff
@@ -25,6 +25,7 @@
 from tempfile import NamedTemporaryFile
 
 import unicodecsv as csv
+from six import string_types
```

Comment:
We don't use six on master branch anymore.

Reply:
Woops, I will remove that! Good catch.